src/Value/Address.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DigipolisGent\Flanders\BasicRegisters\Value;
+
+use DigipolisGent\Value\ValueAbstract;
+use DigipolisGent\Value\ValueInterface;
+
+/**
+ * A single address as used in the Addresses collection.
+ *
+ * This contains only a summary of the address, the full address needs to be
+ * retrieved as AddressDetail.
+ */
+class Address extends ValueAbstract
+{
+    /**
+     * The address object id.
+     *
+     * @var \DigipolisGent\Flanders\BasicRegisters\Value\ObjectId
+     */
+    private $objectId;
+
+    /**
+     * The address house number.
+     *
+     * @var string
+     */
+    private $houseNumber;
+
+    /**
+     * The address bus number.
+     *
+     * @var string
+     */
+    private $busNumber;
```

Comment:
box?

Reply:
I follow the namespace of Flanders basic register: they use everywhere "busnummer".